src/pages/ManagePricing/ManagePricing.module.css

```diff
@@ -1,44 +1,48 @@
-.subtitle {
-  margin-top: 15px;
-  margin-bottom: 15px;  
-  font-style: normal;
-  font-weight: bold;
-  font-size: 14px;
-  line-height: 28px;
-  letter-spacing: 0.02em;
-  color: var(--color-tundora);
-  text-align: center;
-}
-
 .inputs {
   display: flex;
+  margin-top: 20px;
   margin-bottom: 50px;
+  justify-content: space-evenly;
+  align-items: center
 }
 
 .price-input {
-  margin-right: 5px;
   border: 1px solid lightgray;
   text-align:right;
-  font-size: 16px;
-  max-width: 80px;
+  font-size: 18px;
+  text-align: center;
+  padding: 10px;
+  max-width: 30px;
```

Comment:
This width doesn't allow 4 digit numbers. As we can specify float amounts of HF think we'll need a bigger box.